google/cloud/bigquery/types/__init__.py

```diff
@@ -1,5 +1,4 @@
-# -*- coding: utf-8 -*-
-# Copyright 2020 Google LLC
+# Copyright 2021 Google LLC
```

Comment:
Strictly speaking, no. I preserved the structure to avoid unnecessary additional friction when migrating user code from 2x -> 3x. However, it eventually turned out that all that's left in here are the standard SQL types, meaning that the `types` module could indeed be renamed to `standard_sql` - if we want to do it, now is the chance.

User imports will still break, but there's not much difference between renaming `bigquery_v2` to `bigquery` in the import line or creating a slightly different import alias:
```patch
- from google.cloud.bigquery_v2 import types
+ from google.cloud.bigquery import standard_sql as types
```
As an alternative, users can also opt for the plain old find/replace, which requires more or less the same amount of work in both cases.

I think I'll just hoist the `standard_sql` module directly under `bigquery` and that should be it.